package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "express": "^4.18.2"
+  },
+  "devDependencies": {
+    "nodemon": "^3.0.3"
+  },
+  "name": "s47_the_dark_project",
+  "description": "",
+  "version": "1.0.0",
+  "main": "routes.js",
+  "scripts": {
```

Comment:
It would be better to add a start script in the package.json file. This can make it easier to start the application. [medium]